### ISSUE TITLE
azw3 book imported but not decrypted

### ISSUE THREAD

**Issue by PKutz** [apprenticeharper/DeDRM_tools#279]
I installed the DeDRM plugin to Calibre and entered my Kindle serial number. I then went to the Amazon web site and chose the option to download a purchased Kindle book via USB. I then added the book to my Calibre library. It reads the metadata and displays the cover to the book. When I try to open the book in the Calibre reader, I get a message that the book has DRM.

Below is the debug log. It shows that DeDRM started up, but it doesn't seem to have run when I add the book. What am I doing wrong? Please note, I do not have Kindle for PC installed and would prefer not to.

calibre Debug log
calibre 2.85.1 [64bit]  embedded-python: True is64bit: True
Windows-8-6.2.9200 Windows ('64bit', 'WindowsPE')
('Windows', '8', '6.2.9200')
Python 2.7.9
Windows: ('8', '6.2.9200', '', 'Multiprocessor Free')
Successfully initialized third party plugins: DeDRM (6, 5, 3)
devicePixelRatio: 1.0
logicalDpi: 240.0 x 240.0
physicalDpi: 276.462585034 x 277.090909091
Starting up...
Started up in 5.79 seconds with 21 books
stdout+stderr from file dialog helper: ['', '']
piped data from file dialog helper: ['\x0c\xddK\x0e2}\xb4q\xa0c\x1f48S\xc1\x95\xaeJ\x8c\xf4\xa52nc\x01\x16_\n\xc2\x17\xae\x8b', 'C:\\Users\\kutze\\Downloads\\Golden Prey (A Prey Novel).azw3']
Added Golden Prey (A Prey Novel) to db in: 0.3
Added 1 books in 0.9 seconds
Worker Launch took: 0.269999980927

**Comment by ElleKayEm** [comment]
Is this for an E-ink based Kindle?  Not a Fire?

**Comment by PKutz** [comment]
That is correct. It's a Kindle Voyage. I went to the Amazon page to "Manage Your Content and Devices". I then clicked on the little menu icon next to the book and clicked on "Download & transfer via USB". I made sure my Kindle Voyage was selected and the azw3 file downloaded to my Windows 10 PC. I then clicked "Add" in Calibre and selected that file.

**Comment by ElleKayEm** [comment]
Very odd that it doesn't even try to decrypt it.  It sounds like everything should be good.  Look in the plugins area and make sure you didn't accidentally disable DeDRM.

**Comment by PKutz** [comment]
Holy mackerel! You are right. It was indeed disabled!

It wasn't clear to me when clicking the enable/disable button if the plugin was enabled or disabled because the size of the window was to small and the last line wasn't showing. When I enlarged the window, I see the last line when the plug-in is disabled.

I enabled it and it works perfectly. So sorry for my error. Thank you so much for helping me. I appreciate it greatly.
